### DISCUSSION TITLE
Follow up on 7th Sept meeting.

### DISCUSSION THREAD

**Discussion by ianmiell** [finos/common-cloud-controls#36] [General]
Hi @jonmuk @iMichaela,

As discussed in the meeting of 7th September, I'm introducing you to my colleagues @gusfcarvalho and Onsel (invite pending, https://github.com/onselakin).

As a reminder of the context, Onsel and Gustavo are working with the OSCAL standard as part of an open source project we're working on. They've had some lessons learned as a result and we're keen to discuss them with you, as well as hear from you about your perspectives on it.

If you'd like to continue the discussion by email, my email is: ian.miell [at] container-solutions.com (I don't think I have your mails). Anyone else with an interest in this group is welcome to join us.

Ian

**Comment by mcleo-d** [comment]
This discussion can now be found here .. https://github.com/finos/common-cloud-controls/issues/40

**Comment by iMichaela** [comment]
Thank you for introducing Gustavo and Onsel. I would be very happy to learn more about their effort. I sent you an email, so we can connect and start the dialog.
